src/NHibernate/Driver/MicrosoftDataSqlClientDriver.cs

```diff
@@ -98,6 +103,8 @@ public override void Configure(IDictionary<string, string> settings)
 			base.Configure(settings);
 
 			_dialect = Dialect.Dialect.GetDialect(settings);
+
+			_useDbBatch = settings.TryGetValue("usedbbatch", out var value) && bool.Parse(value);
```

Comment:
Do we need this as an option? Users already can configure the batching factory.

Reply:
Removed